R/with.R

```diff
@@ -80,11 +80,11 @@ modify_defaults <- function(defaults, ...) {
 #' # `linters_with_defaults()` and `linters_with_tags("default")` are the same:
 #' all.equal(linters_with_defaults(), linters_with_tags("default"))
 #'
-#' # Get all linters useful for package development
-#' linters_with_tags(tags = "package_development")
+#' # Get names of all linters useful for package development
+#' names(linters_with_tags(tags = "package_development"))
```

Comment:
I think it would be better to store the actual object of interest in a variable and then (maybe) show names(...)